MetaMask/metamask-extension
# Bump dependencies of network-controller

## Explanation



In preparation for introducing `@metamask/network-controller`, we want to ensure that any packages that depend on this package are up to date so that we are using the same version of `@metamask/network-controller` in the dependency tree.

The extension uses two packages that depend on
`@metamask/network-controller`: `@metamask/assets-controllers` and `@metamask/gas-fee-controller`. This commit upgrades them to the latest version. There are no breaking changes in the new release of these packages so there are no accompanying compatibility changes.

Fixes #19099.

## Manual Testing Steps



This is an internal change and should have no user-facing impact. That said, it could affect the list of NFTs/tokens and it could affect gas estimates. So it might be good to double-check that you can add a new token (or that the token autodetection works) and that if you create a transaction you see the gas estimates.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
